bpf/kmesh/workload/sockops_tuple.c

```diff
@@ -155,15 +155,19 @@ static inline bool conn_from_cni_sim_add(struct bpf_sock_ops *skops)
 #if !OE_23_03
     return ((bpf_ntohl(skops->remote_ip4) == 1) && (bpf_ntohl(skops->remote_port) == 0x3a1));
 #else
-    return ((bpf_ntohl(skops->remote_ip4) == 1) && (bpf_ntohl(skops->remote_port) == 0xa1030000));
+    return ((bpf_ntohl(skops->remote_ip4) == 1) && (bpf_ntohl(skops->remote_port) == 0x3a10000));
```

Comment:
Is this only OE2303 behaves like this? And how about the other os releases?

Reply:
Yes, this is a bug in OE2303. However, OE2303 has only a half-year maintenance period. Now OE2303 is no longer updated. Therefore, we need to determine the OE2303 policy and whether to continue the maintenance support.

Reply:
Good to know. We can discuss on community meeting.

But this fix makes sense